README.md

```diff
@@ -1 +1,178 @@
 # ng-bem-components
+
+An [AngularJS](angularjs.org) `angular-bem` module for using components based on [`i-bem.js`](https://bem.info/libs/bem-core/v2.7.0/desktop/i-bem/) framework.
+See [bem-core](https://github.com/bem/bem-core) and [bem-components](https://github.com/bem/bem-components) for further details.
+
+Module will allow you to render `BEMJSON`, help you to work with BEM mods and event, and provide some integrations of `i-bem` blocks with Angular directives.
+
+## Install
+
+```sh
+bower i ng-bem-components --save
+```
+
+`angular-bem` depends on `i-bem__dom`, `jquery`, `BEMHTML` and `BEMTREE` (optional) from `ymodules` modular system.
```

Comment:
https://en.bem.info/libs/bem-core/v2.7.0/desktop/i-bem/jsdoc/#jsdoc-i-bem__dom-1 -> https://en.bem.info/libs/bem-core/current/desktop/i-bem/jsdoc/#jsdoc-i-bem__dom-1

> https://en.bem.info/libs/bem-core/v2.7.0/desktop/jquery/ empty

yeah, translation is still in queue :(

https://en.bem.info/tools/bem/modules/ looks good for me.
BEMTREE — https://en.bem.info/technology/bemtree/
BEMHTML — https://en.bem.info/technology/bemhtml/current/rationale/